### PR TITLE
Correct camera demo error message

### PR DESCRIPTION
**Pull request by AleksandarPetrov**
@afdaniele The error message was wrong. This is the correct one. Tested.

### CODE REVIEW

**Comment by AleksandarPetrov** [comment]
Reviewed by @surirohit. Merging.

File: duckiebot/calibrate_intrinsics/command.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from utils.cli_utils import start_command_in_subprocess
 from utils.docker_utils import get_remote_client, remove_if_running, pull_if_not_exist
 from utils.networking_utils import get_duckiebot_ip
-
+dts duckiebot demo --demo_name camera  --package_name pi_camera --image duckietown/dt-core:daffy --duckiebot_name aidobot11
```
Comment (liampaull) [inline duckiebot/calibrate_intrinsics/command.py]: @AleksandarPetrov this looks like some spurious code that will cause failure no?

Comment (AleksandarPetrov) [reply duckiebot/calibrate_intrinsics/command.py]: @liampaull absolutely, must have copied it by mistake, thanks for catching it!

**Comment by AleksandarPetrov** [comment]
@afdaniele can you review and merge?

**Pull request merged**